### PR TITLE
feat(ingest): Do not use processed tsv but raw jsonl when ingesting data from NCBI Virus

### PR DESCRIPTION
**Pull request by anna-parker**
resolves https://github.com/loculus-project/loculus/issues/1428

preview URL: https://change-how-ingest-ingests.loculus.org/

### Summary

Prompted by https://github.com/loculus-project/loculus/pull/2986#issuecomment-2407406053. The tsv generated by the ncbi datasets cli from the raw jsonl does not correctly format authors- leading to the case where the string cannot be correctly parsed. To avoid this I change the ingest code to use the raw jsonl directly. This way we can format authors correctly, and have access to the raw data. 

### Fields that existed in the (renamed) tsv and not in the jsonl

Of these fields only two were actually mapped to Loculus fields. The rest were all dropped by ingest - so not having them in the jsonl should not be a problem

```
ncbiHostBreed
ncbiHostCultivar
ncbiHostEcotype
ncbiHostIsolate
ncbiHostPangolin
ncbiIsVaccineStrain
ncbiMaturePeptideCount
ncbiMolType
ncbiVirusCommonName
ncbiVirusBreed
ncbiVirusCultivar
ncbiVirusEcotype
ncbiVirusIsolate
ncbi_virus (listed as "Virus Infraspecific Names Sex" in column_mapping)
ncbiVirusStrain
ncbiVirusPangolin
ncbiHostCommonName
ncbiPurposeOfSampling
```

### Missing fields that were mapped to Loculus fields

```
ncbiHostCommonName
ncbiPurposeOfSampling
```

### Potentially useful fields (that are currently dropped

```

ncbiIsVaccineStrain
```

### Testing

Checkout https://github.com/pathoplexus/pathoplexus/commit/ce68b92e8b01055a36b5fa60893876078963bf98 to perform regression testing (ignore authors as should be different) by comparing this branch with Loculus main: 

```sh
micromamba activate pp-integrity
cd tests/regression-testing 
snakemake results/ebola-zaire.meta.main.change-how-ingest-ingests.diff 
snakemake results/cchf.meta.main.change-how-ingest-ingests.diff 
snakemake results/west-nile.meta.main.change-how-ingest-ingests.diff 
```

There is no diff in the ebola-zaire, cchf and west nile metadata.

### Additional changes

Remove requirement that authors list ends with ";". However warn users if authors list might be in incorrect format.
Raise error if sequence is not in correct format or not ASCII and give users error message with correct format.
In ingest if authors list is entirely capitalized apply title case (leave common last names prefixes such as van etc. lower case)
Add ingest test by modifying snakemake example files to also have edge cases.

### PR Checklist

- [x] All necessary documentation has been adapted.
- [x] The implemented feature is covered by an appropriate test.

### CODE REVIEW

File: ingest/scripts/rename_fields.py
Comment (anna-parker) [inline ingest/scripts/rename_fields.py:1]: This will need to be tidied up I just manually mapped the fields of the json dict to the old tsv output using only the CCHF dataset as an example. Then I had chatgpt write me this script to manually map these fields and convert them to a tsv.

File: ingest/config/defaults.yaml [hunk 1]
```diff
@@ -24,52 +24,40 @@ keep:
...
+  - "ncbiIsolateName": "name"
+  - "ncbiIsolateSource": "source"
+  - "ncbiCollectionDate": "collectionDate"
+last_virus_lineage_mappings:
+  - "ncbiVirusName": "name"
+last_host_lineage_mappings:
+  - "ncbiHostTaxId": "taxon_id"
+  - "ncbiHostName": "name"
+  - "ncbiHostSex": "sex"
+unknown_mappings: # I don't know yet where these fields come from
```
Comment (anna-parker) [inline ingest/config/defaults.yaml:58]: @corneliusroemer I am not sure where these metadata fields would be in the jsonl

**Comment by corneliusroemer** [comment]
Immediate thoughts from reading PR (not looking at code yet):

- I'm confused by the long list of  "Fields that existed in the tsv and not in the jsonl" - are you sure this is correct? I don't think dataformat does much other than reformat the jsonl - but maybe I'm wrong.

**Comment by anna-parker** [comment]
Yes - I am almost 100% sure that the list in "Fields that existed in the tsv and not in the jsonl" is correct - I was constantly comparing the output of my tsv files. I think datasets creates columns for data fields that exist is other datasets even if they do not exist in our own dataset (alternatively it is better at mapping the list of clade names to metadata fields).

**Comment by theosanderson** [comment]
If one wanted to figure out where these come from they'd likely feature in a SC2 dataset
